pkg/api/v1/client/factory.go

```diff
@@ -10,19 +12,19 @@ const (
 )
 
 func GetClient(clientType ClientType, namespace string) (client Client, err error) {
-	switch clientType {
-
-	case ClientDirect:
-		client = NewDefaultDirectAPIClient()
-	case ClientProxy:
-		clientset, err := GetClientSet()
-		if err != nil {
-			return client, err
-		}
-		client = NewProxyAPIClient(clientset, NewProxyConfig(namespace))
-	default:
-		err = fmt.Errorf("Client %s is not handled by testkube, use one of: %v", clientType, []ClientType{ClientDirect, ClientProxy})
+	var overrideHost string
+	var clientset kubernetes.Interface
+
+	if clientType == ClientDirect {
+		overrideHost = "http://127.0.0.1:8080"
```

Comment:
I would make it a const or env var

Reply:
done (added possibility to override by env var